addons/mrp_subcontracting/models/stock_picking.py

```diff
@@ -131,16 +131,19 @@ def _prepare_subcontract_mo_vals(self, subcontract_move, bom):
         })
         product = subcontract_move.product_id
         warehouse = self._get_warehouse(subcontract_move)
+        company_id = subcontract_move.company_id
+        picking_id = subcontract_move.picking_id
+        subcontracting_location_id = picking_id.partner_id.with_company(company_id).property_stock_subcontractor or company_id.subcontracting_location_id
         vals = {
-            'company_id': subcontract_move.company_id.id,
+            'company_id': company_id.id,
```

Comment:
```suggestion
            'company_id': subcontract_move.company_id.id,
```
Let's avoid useless diff

Reply:
> Let's avoid useless diff

Hello, The code has been updated according to your suggestion. Thanks :)